module/apmhttp/ignorer.go

```diff
@@ -53,6 +54,16 @@ func DefaultServerRequestIgnorer() RequestIgnorerFunc {
 	return defaultServerRequestIgnorer
 }
 
+// DynamicServerRequestIgnorer returns the RequestIgnorer to use in
```

Comment:
I think we should add a deprecation comment to DefaultServerRequestIgnorer above, seeing as it confusingly will no longer name have an apt name "Default..."